src/host/cmdline.cpp

```diff
@@ -198,7 +196,8 @@ void CommandLine::EndAllPopups()
 {
     while (!_popups.empty())
     {
-        EndCurrentPopup();
+        _popups.front()->End();
+        _popups.pop_front();

```

Comment:
Why?

Reply:
Yeah, I'd prefer to keep this as `EndCurrentPopup()`. Theoretically, if we changed `EndCurrentPopup()` (which is unlikely) this would be stuck with the older implementation.